Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/eventregistration/organizer/QrCodeDetailFragmentTest.kt

```diff
@@ -76,8 +76,8 @@ class QrCodeDetailFragmentTest : BaseUITest() {
             ).toBundle()
         )
 
-        onView(withId(R.id.title)).check(matches(withText("Jahrestreffen der deutschen SAP Anwendergruppe")))
-        onView(withId(R.id.subtitle)).check(matches(withText("Hauptstr. 3, 69115 Heidelberg")))
+        onView(withId(R.id.tl_organizer_title)).check(matches(withText("Jahrestreffen der deutschen SAP Anwendergruppe")))
+        onView(withId(R.id.tl_organizer_subtitle)).check(matches(withText("Hauptstr. 3, 69115 Heidelberg")))
         onView(withId(R.id.eventDate)).check(matches(withText("21.06.2021, 18:00 - 21:00 Uhr")))
```

Comment:
Maybe name these with the full name: `trace_location_organizer_title` to have consistencty.